util/artifacts.go

```diff
@@ -63,17 +63,29 @@ func streamArchiveContents(ctx context.Context, rootPath string, includes, exclu
 
 			if filematch == "**" {
 				walk = func(path string, info os.FileInfo, err error) error {
+					for _, ignore := range excludes {
+						if match, _ := filepath.Match(ignore, path); match {
+							return nil
+						}
+					}
+
 					outputChan <- ArchiveContentFile{path, info, nil}
-					return
+					return nil
```

Comment:
Do you really want the walk to continue? I had fixed the compile by returning `err`.

Reply:
I think so, there's no error in this case, and the archive content collector has continue on error semantics, so in cases where there are errors they get propagated [eg](https://github.com/evergreen-ci/evergreen/pull/1280/files#diff-ac23dd138157b5221a0eaa4a257333e9R97).